metadata/elasticsearch/src/main/java/com/spotify/heroic/metadata/elasticsearch/MetadataBackendKV.java

```diff
@@ -209,7 +211,23 @@ public Groups groups() {
                 .startSpan();
             final Scope rootScope = tracer.withSpan(rootSpan);
 
-            final Series series = request.getSeries();
+            final Series series;
```

Comment:
As discussed offline we could optimize it a bit and add code to `buildContext` method to add resources to ES request.
Something like this

```java
private static void buildContext(final XContentBuilder b, Series series, boolean indexResourceIdentifiers) throws IOException {
        b.field(KEY, series.getKey());

        b.startArray(TAGS);
        for (final Map.Entry<String, String> entry : series.getTags().entrySet()) {
            b.value(entry.getKey() + TAG_DELIMITER + entry.getValue());
        }

        if(indexResourceIdentifiers) {
           for (final Map.Entry<String, String> entry : series.getResource().entrySet()) {
               b.value(entry.getKey() + TAG_DELIMITER + entry.getValue());
           }
        }

        b.endArray();

        b.startArray(TAG_KEYS);
        for (final Map.Entry<String, String> entry : series.getTags().entrySet()) {
            b.value(entry.getKey());
        }
        b.endArray();

        b.field(HASH_FIELD, series.hash());
    }